guides/common/modules/proc_creating-a-remote-source-for-a-custom-file-type-repository.adoc

```diff
@@ -90,7 +90,7 @@ PULP_MANIFEST test.txt
 ----
 
 Now, you can import your remote source as a {customfiletype} repository.
-Use the path to the directory to specify an *Upstream URL*, such as `\http://{foreman-example-com}/__my_file_repo__`.
+Use the path to the directory to specify an *Upstream URL*, such as `\http://{server-example-com}/__my_file_repo__`.
```

Comment:
This was my only intended change. The procedure is for a "remove source", so using foreman.example.com does not make any sense.